examples/typescript/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+    "compilerOptions": {
+        "module": "esnext",
+        "jsx": "react",
```

Comment:
Should TypeScript compile JSX?

Reply:
Yeah so there are technically 2 approaches. Compile TypeScript inline once or replcae babel-loader with awesome-typescript-loader / ts-loader. Since after is basically razzle under the hood, should be able to copy and paste from the Razzle TypeScript example